src/components/Call.js

```diff
@@ -5,21 +5,34 @@ class Call extends React.Component {
     render() {
         var styling = "";
         var dataTarget = "#exportModal";
-        if (this.props.empty) {
-            styling = "disabled list-group-item-light call-empty";
-            dataTarget = "";
-        } else if (this.props.disabled) {
-            styling = "list-group-item-dark call-disabled";
+        var callStyle;
+        if (this.props.draggable) {
+            styling = "inherit-rounded";
         } else {
-            styling = `btn-outline-light group-${this.props.group}`;
+            callStyle = {height: `${this.props.callSize.height}px`, width: `${this.props.callSize.width}px`};
```

Comment:
Moved to another method, although the exact line commented on was moved rather than changed so the requested change didn't automatically resolve itself